docs/_src/language/basic.md

```diff
@@ -164,7 +164,7 @@ explore 'malloy-data.faa.airports'
 ## Timeframes and Timezones.
 Time is a big deal in data.  Malloy has built in contructs to easily handle time, relative time filtering, date ranges and time stamps.  We'll write more about this when the syntax is finalized.
 
-## Turtles
+## Aggregating Subqueries
 In malloy queries can nest in other queries.
```

Comment:
In malloy queries can be nested to produce subtables on each output row of a query.